archunit-example/example-plain/src/main/java/com/tngtech/archunit/example/cycle/complexcycles/slice1/UnrelatedEnum.java

```diff
@@ -0,0 +1,14 @@
+package com.tngtech.archunit.example.cycle.complexcycles.slice1;
+
+// This was just added for the integration tests since it creates some synthetic byte code
+@SuppressWarnings("SwitchStatementWithTooFewBranches")
+public enum UnrelatedEnum {
```

Comment:
Probably indeed unrelated to the package name of array types... 😉

Reply:
Yeah, my mind might have jumped a bit :wink: Thought before I completely forget I better add it on the fly...